templates/static/js/example.js

```diff
@@ -9,6 +28,16 @@
     }
   });
 
+  document.addEventListener('click', function (e) {
```

Comment:
Why not target `js-codepen-link` class instead of adding a listener to the body?

Reply:
I'm fairly certain it used to be difficult to attach events to dynamically created elements, and this was a workaround - turns out that's not the case..! I've attached the event at the time the link is appended, removing this block altogether.